README.md

```diff
@@ -20,3 +20,20 @@ See [this list](https://www.emacswiki.org/emacs/FixmeMode) on the Emacswiki for
 other packages that implement the same basic features, but which might
 also provide additional features that you might like, but which I
 don't deem necessary.
+
+Customizing Tags
+================
+
+You can customize the code tags that are recognized by hl-todo by
+customizing `hl-todo-keyword-faces` either through the customization
+menu or by setting the variable directly in your `.emacs` file for
```

Comment:
Use "your init file" instead of "your `.emacs` file". Nowadays almost nobody uses an init file by the name `.emacs`.